README.md

```diff
@@ -5,13 +5,14 @@ WSO2 Balana Implementation
 
 |  Branch | Build Status |
 | :------------ |:-------------
-| master      | [![Build Status](https://wso2.org/jenkins/job/wso2-balana/badge/icon)](https://wso2.org/jenkins/job/wso2-balana) |
-
+| master      | [![Build Status](https://wso2.org/jenkins/buildStatus/icon?job=forked-dependencies/wso2-balana)](https://wso2.org/jenkins/job/forked-dependencies/wso2-balana) |
```

Comment:
Build status badge seems to be working. But the link for the jenkins build should be https://wso2.org/jenkins/job/forked-dependencies/job/wso2-balana/